### PR TITLE
Alter log level for disabled services

### LINKED ISSUE #1418: Panic within GCP redis calculating costs
**Describe the bug**
```
panic: runtime error: index out of range [0] with length 0

goroutine 48 [running]:
github.com/tailwarden/komiser/providers/gcp/redis.calculateRedisCost(0xc0009623c0, 0xc00268a540?)
	/home/runner/work/komiser/komiser/providers/gcp/redis/pricing.go:79 +0x365
github.com/tailwarden/komiser/providers/gcp/redis.Instances({0x4231180, 0x6504b20}, {0x0, 0x0, {0x0, 0x0}, 0x0, 0x0, 0x0, 0x0, ...})
	/home/runner/work/komiser/komiser/providers/gcp/redis/instances.go:68 +0x714
github.com/tailwarden/komiser/providers/gcp.FetchResources.func1()
	/home/runner/work/komiser/komiser/providers/gcp/gcp.go:57 +0x99
github.com/tailwarden/komiser/providers.(*WorkerPool).worker(0xc000d9e000)
	/home/runner/work/komiser/komiser/providers/providers.go:85 +0x3a
created by github.com/tailwarden/komiser/providers.(*WorkerPool).Start in goroutine 50
	/home/runner/work/komiser/komiser/providers/providers.go:69 +0x25
```

**To Reproduce**
1. I run docker image with config.toml set to a GCP account
2. it started, run fetching some objects apparently for a while and then died with message above

**Expected behavior**
- no panic during fetching objects and doing calculations

**Desktop (please complete the following information):**
 - OS: Ubuntu/Docker
 - Browser - it fails without any interaction from UI
 - Version: 
 ```
user@laptop:~$ docker images
REPOSITORY           TAG       IMAGE ID       CREATED       SIZE
tailwarden/komiser   latest    894b195df7ac   10 days ago   150MB
```

### PR DESCRIPTION
**Pull request by Azanul**
## Problem

Issue #1418 

## Solution

Fixes #1418 

## Changes Made

- GCP services that are disabled will show a warning instead of an error log
- Fixed bucket size calculation

## Checklist

- [X] Code follows the <a href="https://github.com/tailwarden/komiser/blob/master/CONTRIBUTING.md">contributing</a> guidelines
- [X] Changes have been thoroughly tested
- [X] <a href="https://github.com/tailwarden/docs.komiser.io">Documentation</a> has been updated, if necessary
- [X] Any dependencies have been added to the project, if necessary

### CODE REVIEW

File: providers/gcp/gateway/gateways.go [hunk 1]
```diff
@@ -34,12 +35,12 @@ RegionsLoop:
 			"projects/" + client.GCPClient.Credentials.ProjectID + "/locations/" + regionName,
 		).Do()
 		if err != nil {
-			if err.Error() == "googleapi: Error 403: Location "+regionName+" is not found or access is unauthorized., forbidden" {
+			if err.Error() == "googleapi: Error 403: Location "+regionName+" is not found or access is unauthorized., forbidden" || strings.Contains(err.Error(), "SERVICE_DISABLED") {
```
Comment (AvineshTripathi) [inline providers/gcp/gateway/gateways.go:38]: is this resource specific or general 403 for all the services

**Review by AvineshTripathi — COMMENTED** [review]
I would prefer a generic error handler that checks if the error with provider exist in our list and ignore it(this could stay in provider util- a provider specific handler)

File: providers/gcp/gateway/gateways.go (hunk 1 above)
Comment (Azanul) [reply providers/gcp/gateway/gateways.go:38]: I'm not sure, it was there already so I didn't change it

**Comment by Azanul** [comment]
> I would prefer a generic error handler that checks if the error with provider exist in our list and ignore it(this could stay in provider util- a provider specific handler)

Good suggestion! I thought so too but currently we have only one such error. Will implement it once we have more

**Review by AvineshTripathi — APPROVED** [review]
we should create an issue for similar cases for different provider

**Comment by Azanul** [comment]
> we should create an issue for similar cases for different provider

This service disabled errors are not actual errors was pointed out by a user. If you have knowledge of such cases for any providers please feel free to create respective issues

**Pull request merged**